<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchMenu</class>
 <widget class="QDialog" name="SearchMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/icons.qrc">
    <normaloff>:/monokuma-green.png</normaloff>:/monokuma-green.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="txtQuery">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSearch">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Search</string>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeResults">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>283</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="indentation">
          <number>15</number>
         </property>
         <column>
          <property name="text">
           <string>Results (0)</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <family>Meiryo UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Translated</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="txtTranslated">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Meiryo UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::WidgetWidth</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="plainText">
          <string/>
         </property>
         <property name="tabStopWidth">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <family>Meiryo UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Original</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="txtOriginal">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Meiryo UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::WidgetWidth</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="plainText">
          <string/>
         </property>
         <property name="tabStopWidth">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="font">
          <font>
           <family>Meiryo UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Comments</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="txtComments">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Meiryo UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::WidgetWidth</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="plainText">
          <string/>
         </property>
         <property name="tabStopWidth">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="grpAdvanced">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Meiryo UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Advanced</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <widget class="QCheckBox" name="chkAdvCase">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Case Sensitive</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>181</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Search</string>
        </property>
        <widget class="QCheckBox" name="chkAdvTrans">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>18</y>
           <width>91</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Translated</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="chkAdvOrig">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>38</y>
           <width>70</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Original</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="chkAdvComments">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>58</y>
           <width>91</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Comments</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
       <widget class="QCheckBox" name="chkAdvRegex">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Regex Search</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>181</width>
          <height>141</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Search Filter</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>94</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Filter Regex</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="txtFilterRe">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>114</y>
           <width>161</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>161</width>
           <height>71</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="layoutSearchFilter">
          <item row="1" column="0">
           <widget class="QCheckBox" name="chkSearchCh3">
            <property name="text">
             <string>CH3</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="chkSearchPlg">
            <property name="text">
             <string>PLG</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="chkSearchCh1">
            <property name="text">
             <string>CH1</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="chkSearchCh2">
            <property name="text">
             <string>CH2</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="chkSearchCh4">
            <property name="text">
             <string>CH4</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="chkSearchCh5">
            <property name="text">
             <string>CH5</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="chkSearchCh6">
            <property name="text">
             <string>CH6</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="chkSearchEpg">
            <property name="text">
             <string>EPG</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="chkSearchFt">
            <property name="text">
             <string>FT</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="chkSearchEtc">
            <property name="text">
             <string>ETC</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="chkSearchSys">
            <property name="text">
             <string>SYS</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QPushButton" name="btnFilterSelAll">
        <property name="geometry">
         <rect>
          <x>107</x>
          <y>140</y>
          <width>48</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Select All</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnFilterSelNone">
        <property name="geometry">
         <rect>
          <x>155</x>
          <y>140</y>
          <width>32</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>None</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="chkAdvNewline">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>20</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>w/Newlines</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="chkAdvNoTags">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>40</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ignore CLT</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Open</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtQuery</tabstop>
  <tabstop>btnSearch</tabstop>
  <tabstop>treeResults</tabstop>
  <tabstop>chkAdvRegex</tabstop>
  <tabstop>chkAdvCase</tabstop>
  <tabstop>chkAdvTrans</tabstop>
  <tabstop>chkAdvOrig</tabstop>
  <tabstop>chkAdvComments</tabstop>
  <tabstop>btnFilterSelAll</tabstop>
  <tabstop>btnFilterSelNone</tabstop>
  <tabstop>chkSearchPlg</tabstop>
  <tabstop>chkSearchCh1</tabstop>
  <tabstop>chkSearchCh2</tabstop>
  <tabstop>chkSearchCh3</tabstop>
  <tabstop>chkSearchCh4</tabstop>
  <tabstop>chkSearchCh5</tabstop>
  <tabstop>chkSearchCh6</tabstop>
  <tabstop>chkSearchEpg</tabstop>
  <tabstop>chkSearchFt</tabstop>
  <tabstop>chkSearchSys</tabstop>
  <tabstop>chkSearchEtc</tabstop>
  <tabstop>txtFilterRe</tabstop>
  <tabstop>txtTranslated</tabstop>
  <tabstop>txtOriginal</tabstop>
  <tabstop>txtComments</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../res/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SearchMenu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SearchMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSearch</sender>
   <signal>clicked()</signal>
   <receiver>SearchMenu</receiver>
   <slot>search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeResults</sender>
   <signal>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</signal>
   <receiver>SearchMenu</receiver>
   <slot>changedSelection(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeResults</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>SearchMenu</receiver>
   <slot>doubleClicked(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSearchPlg</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SearchMenu</receiver>
   <slot>changedSearchFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>632</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSearchCh1</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SearchMenu</receiver>
   <slot>changedSearchFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>754</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSearchCh2</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SearchMenu</receiver>
   <slot>changedSearchFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>796</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>632</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSearchCh3</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SearchMenu</receiver>
   <slot>changedSearchFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>680</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSearchCh4</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SearchMenu</receiver>
   <slot>changedSearchFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>634</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSearchCh5</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SearchMenu</receiver>
   <slot>changedSearchFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>780</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>634</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSearchCh6</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SearchMenu</receiver>
   <slot>changedSearchFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>686</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>631</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSearchEpg</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SearchMenu</receiver>
   <slot>changedSearchFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>632</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSearchFt</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SearchMenu</receiver>
   <slot>changedSearchFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>773</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>632</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSearchEtc</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SearchMenu</receiver>
   <slot>changedSearchFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>719</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>621</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSearchSys</sender>
   <signal>stateChanged(int)</signal>
   <receiver>SearchMenu</receiver>
   <slot>changedSearchFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>659</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>search()</slot>
  <slot>changedSelection(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
  <slot>doubleClicked(QTreeWidgetItem*,int)</slot>
  <slot>changedSearchFilter()</slot>
 </slots>
</ui>
